deepchem/feat/bert_featurizer.py

```diff
@@ -0,0 +1,65 @@
+from deepchem.feat.base_classes import Featurizer
+from deepchem.feat.molecule_featurizers import RobertaFeaturizer
+from typing import Iterable
+import numpy as np
+import logging, warnings
+logger = logging.getLogger(__name__)
+try:
+  from transformers import BertTokenizerFast, RobertaTokenizerFast
+except:
+  raise ImportError("""This class requires the transformers package,
+                    which was not found in your environment.""")
+# TODO discuss RDKitMol support (perhaps RobertaFeaturizer should not be a moleule featurizer?
+# Current approach to handling both molecules and strings is a bit messy.
+try:
+  from deepchem.utils.typing import RDKitMol
+except:
+  warnings.warn("""RDKitMol was not found in your environment, so this class
+                will only featurize strings and will not featurize RDKitMol
+                objects.""")
+                
+
+class BertFeaturizer(RobertaFeaturizer, BertTokenizerFast):
+  """Bert Featurizer.
+
+  The Bert Featurizer is a wrapper class for HuggingFace's BertTokenizerFast.
+
+  This class intends to allow users to use the BertTokenizer API while
+  remaining inside the DeepChem ecosystem.
+
+  Usage Example (based on HuggingFace and RostLab ProtBert docs)
+  --------------------------------------------------------------
+  >>> from deepchem.feat import BertFeaturizer
+  >>> featurizer = BertFeaturizer.from_pretrained("Rostlab/prot_bert", do_lower_case=False)
+  >>> inputs = featurizer.featurize('D L I P [MASK] L V T', return_tensors="pt")
+
+  Notes
+  -----
+  This class inherits first from RobertaFeaturizer and then from
+  BertTokenizerFast. Its logic relies on super() calls in RobertaFeaturizer
+  being delegated to BertTokenizerFast in accordance with Python's multiple
+  inheritance rules.
+
+  Parts of this class, including its documentation and naming scheme, is based
+  on and/or bear similarities to RobertaFeaturizer for the sake of consistency.
+  """
+
+  def _featurize(self, sequence: Union[str, RDKitMol]) -> List[List[int]]:
```

Comment:
Similar to the discussion in @seyonechithrananda's PR, let's just make this expect strings? The union type here feels a bit awkward

Reply:
Hmm, that makes sense!

I do want to maintain some consistency with the existing molecular featurizer workflows that expect RDKit molecules, but is that less important for the time being?